packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -195,7 +195,7 @@ export type Actions =
   | NotificationServicesControllerDeleteNotificationsById;
 
 // Allowed Actions
-export type AllowedActions =
+type AllowedActions =
```

Comment:
Because `@metamask/notification-services-controller` is using barrel exports in `index.ts`, I believe that if we don't export this type it will remove the export from the whole package, can you confirm? If so should we add this to the changelog?